Dockerfile-ubuntu2004

```diff
@@ -24,7 +24,10 @@ RUN apt-get update && apt-get -y install \
   wget \
   xz-utils \
   net-tools \
-  tcpreplay
+  tcpreplay \
+  python3 \
```

Comment:
I think `python3` is already installed in all Ubuntu images 🤔

Reply:
True, let me remove it.